rust-sdk/whirlpool/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "3.1.0",
   "scripts": {
     "build": "cargo build",
-    "test": "cargo test --lib",
+    "test": "cp ../../legacy-sdk/whirlpool/tests/external_program/token_2022.20250510.so ../../target/deploy/ && cargo test --lib",
```

Comment:
built-in Token-2022 is too old, so we need to inject newer one into test environment.